index.js

```diff
@@ -0,0 +1,102 @@
+const express = require("express");
+const jwt = require("jsonwebtoken");
+const bcyrpt = require("bcryptjs");
+
+const db = require("./data/dbConfig");
+const configureMiddleware = require("./middleware");
+
+const secret = "BIGBOB454624EEE54TT54efg445GHJ";
+
+const server = express();
+const PORT = 3300;
+
+configureMiddleware(server);
+
```

Comment:
The code here is strong Rob. Your creating your JWT's and verifying/attaching it to a logged in user so they _can_ access restricted parts of the app (showing all the users in this case) 👍